hooks/pre-commit

```diff
@@ -6,7 +6,7 @@ else
     VERSION=$(python3 -c 'import sys; v = sys.version_info; print("{}.{}".format(v[0], v[1]));')
 fi
 python3 -m flake8 -j auto
-if [ $VERSION = '3.4' ]; then
+if [ $VERSION != '3.3' ]; then
```

Comment:
@miaekim If py36, 37, 38.... will release, our code must be `$VERSION = '3.4' or $VESRION = '3.5' or $VESRION = '3.6' or $VESRION = '3.7' or $VESRION = '3.8' or ...`.